src/lib/components/CopyCode/styles.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../../styles/settings/colors";
+@import '../../../styles/settings/global';
+
+.w-copy-code-button {
```

Comment:
Apologies for being a bit inconsistent with our naming.

For custom elements, we always prefix with `web-*`.
For other CSS components we just use the `w-*` namespace.

I realize that's hecka weird but we should be consistent until we update everything. I think we can name this component `web-copy-code` for now, and when I get back the week after next we can do a bit of house cleaning and give everything the same prefix.